tools/migrate_pmid_to_uid/README.md

```diff
@@ -0,0 +1,46 @@
+## migrate_pmid_to_uid
```

Comment:
It is a "done once, never again" migration.

However, what we really need is some sort of overall changelog that would detail these things for anyone with their own setup. Not specific per repo, I guess, as that would most likely not be seen. Basically, some reliable way to communicate with anyone using the code.